External/FEXCore/Source/Utils/AllocatorOverride.cpp

```diff
@@ -53,6 +53,16 @@ namespace FEXCore::Allocator {
   // Enable or disable allocation faulting per-thread.
   static thread_local uint64_t SkipEvalForThread{};
 
+  using MALLOC_Hook = void *(*)(size_t);
+  using CALLOC_Hook = void *(*)(size_t, size_t);
+  using MEMALIGN_Hook = void *(*)(size_t, size_t);
+  using VALLOC_Hook = void *(*)(size_t);
+  using POSIX_MEMALIGN_Hook = int (*)(void**, size_t, size_t);
+  using REALLOC_Hook = void *(*)(void*, size_t);
+  using FREE_Hook = void(*)(void*);
+  using MALLOC_USABLE_SIZE_Hook = size_t (*)(void*);
+  using ALIGNED_ALLOC_Hook = void *(*)(size_t, size_t);
+
   // Internal memory allocation hooks to allow non-faulting allocations through.
   CALLOC_Hook calloc_ptr = __libc_calloc;
```

Comment:
Can we just use `auto` to declare these? Then we don't need most of the extra typedefs above.

Reply:
You're right. Switched over to auto for these.